src/components/app/data/queries/extractEnterpriseCustomer.test.js

```diff
@@ -12,7 +17,9 @@ const mockEnterpriseCustomer = enterpriseCustomerFactory();
 const mockEnterpriseCustomerUser = enterpriseCustomerUserFactory({
   enterprise_customer: mockEnterpriseCustomer,
 });
-
+const mockRequestUrl = {
+  pathname: mockEnterpriseCustomer.slug,
+};
```

Comment:
nit: Might consider using the expected type of `URL` to construct this so TypeScript is happy if/when this file is converted to TypeScript, e.g.

```ts
const mockRequestUrl = new URL(`/${mockEnterpriseCustomer.slug}`, 'http://localhost');
```